x-pack/platform/plugins/private/reporting/server/routes/common/request_handler/lib/transform_raw_scheduled_report.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObject } from '@kbn/core/server';
+import { ScheduledReportApiJSON, ScheduledReportType } from '../../../../types';
+
+export function transformRawScheduledReportToReport(
+  rawScheduledReport: SavedObject<ScheduledReportType>
+): ScheduledReportApiJSON {
+  const parsedPayload = JSON.parse(rawScheduledReport.attributes.payload);
+  return {
+    id: rawScheduledReport.id,
+    jobtype: rawScheduledReport.attributes.jobType,
+    created_at: rawScheduledReport.attributes.createdAt,
+    created_by: rawScheduledReport.attributes.createdBy as string | false,
+    payload: parsedPayload,
+    meta: rawScheduledReport.attributes.meta,
+    migration_version: rawScheduledReport.attributes.migrationVersion,
```

Comment:
I can't tell what this is used for, and wonder how it differs from the base SO `migration_version` field.  I guess the `report` docs have this, as some kind of "follow the SO pattern" since they aren't SO's - I assume we probably need this since the core reporting stuff does, but feels like it may get confusing ...

Reply:
AFAICT, this is always set to 7.14.0 so maybe it's something that's not needed anymore? I was trying to save everything that's currently saved and read for the report generation. I can make it optional in the schema so if we remove it in the future we're not forced to populate it.

Reply:
Updated to make migration version optional in SO schema: https://github.com/elastic/kibana/pull/219771/commits/b6af77043838059fb8cc8417f473fa0010466dc2